pkgdb/src/realisepkgs/realisepkgs-lockfile.cc

```diff
@@ -180,6 +181,12 @@ realisepkgsPackageFromV1Descriptor( const nlohmann::json &     jfrom,
             nix::fmt( "locked_url is not nixpkgs... skipping input for %s ",
                       attrPath ) );
         }
+
+      pkg.outputsToOutpaths = std::make_shared<OutputsToOutpaths>();
+      for ( auto [output, outpath] : jfrom["outputs"].items() )
+        {
+          pkg.outputsToOutpaths.get()->emplace( output, outpath );
```

Comment:
nit: would it be more consistent to not rename this `outputs` -> `outputsToOutpaths`?

Reply:
This is from the lockfile schema I think, so I'll defer that for now.  I think we'll have a lockfile update (maybe?) to account for missing fields for the various package types.  I do agree though.